slides/chapter4_01_training-updating-models.md

```diff
@@ -123,7 +123,7 @@ Because the entity recognizer predicts entities _in context_, it also needs to
 be trained on entities _and_ their surrounding context.
 
 The easiest way to do this is to show the model a text and a list of character
-offsets. For example, "iPhone X" is a gadget, starts at character 0 and ends at
+offsets. For example, "iPhone X" is coming, starts at character 0 and ends at
```

Comment:
Ah, I just realised that this was kind of confusing! What I originally wanted to say here was that the string "iPhone" has the label `GADGET` etc. So maybe it could say: 

> For example, the string "iPhone X" starts at character 0, ends at character 8 and should have the label "gadget".

Reply:
Ah! I get it. I was misinterpreting it, I understand it now.

Given that, any suggestion I make here would be purely subjective, I don't think this is necessarily worth the change.

I'm reverting it to keep the PR purely about the other mini-typos so that you can check if it's actually worth any change here and how later. But I don't see anything wrong with it now that I understand it.